src/MusicPlayer.cpp

```diff
@@ -14,4 +15,17 @@ void MusicPlayer::Toggle() {
     else {
         music.play();
     }
-}
\ No newline at end of file
+}
+void MusicPlayer::Play() {
+    if (music.getStatus() == sf::SoundSource::Status::Paused) {
+        music.play();
+    }
+}
+
+void MusicPlayer::Pause()      
+{        
+    if (music.getStatus() == sf::SoundSource::Status::Playing) {
```

Comment:
Unnecessary check - if you look at the reference (https://www.sfml-dev.org/documentation/2.5.1/classsf_1_1SoundStream.php#a932ff181e661503cad288b4bb6fe45ca) you can see, that calling pause when the music is already paused does nothing. So you can safely just call it without "if"